server/src/test/java/io/crate/execution/engine/aggregation/impl/PercentileAggregationTest.java

```diff
@@ -103,14 +103,14 @@ public void testReturnTypes() throws Exception {
     }
 
     @Test
-    public void testSignleFractionAllTypesReturnSameResult() throws Exception {
+    public void testSingleFractionAllTypesReturnSameResult() throws Exception {
         for (DataType<?> valueType : DataTypes.NUMERIC_PRIMITIVE_TYPES) {
             List<Double> fractions = Arrays.asList(0.5, 0.8);
             Object[][] rowsWithSingleFraction = new Object[10][];
             for (int i = 0; i < rowsWithSingleFraction.length; i++) {
                 rowsWithSingleFraction[i] = new Object[]{ valueType.sanitizeValue(i), fractions.get(0) };
             }
-            assertThat(execSingleFractionPercentile(valueType, rowsWithSingleFraction)).isEqualTo(4.5);
+            assertThat(execSingleFractionPercentile(valueType, rowsWithSingleFraction)).isEqualTo(5.0);
```

Comment:
Looks like a new version of the dependency does rounding up - do we consider this as a breaking change?

Reply:
It doesn't round. The values here are 0.0 - 9.0 with 1.0 increments. So 5.0 is part of the data:

```
Object[10][]@43
  0: Object[2]@71
    0: Double@82 "0.0"
    1: Double@83 "0.5"
  1: Object[2]@72
    0: Double@91 "1.0"
    1: Double@83 "0.5"
  2: Object[2]@73
    0: Double@94 "2.0"
    1: Double@83 "0.5"
  3: Object[2]@74
    0: Double@97 "3.0"
    1: Double@83 "0.5"
  4: Object[2]@75
    0: Double@100 "4.0"
    1: Double@83 "0.5"
  5: Object[2]@76
    0: Double@103 "5.0"
    1: Double@83 "0.5"
  6: Object[2]@77
    0: Double@106 "6.0"
    1: Double@83 "0.5"
  7: Object[2]@78
    0: Double@109 "7.0"
    1: Double@83 "0.5"
  8: Object[2]@79
    0: Double@112 "8.0"
    1: Double@83 "0.5"
  9: Object[2]@80
    0: Double@115 "9.0"
    1: Double@83 "0.5"
```

The old version interpolated, which was kinda wrong. See also https://github.com/opensearch-project/OpenSearch/pull/3634

It's somewhere inbetween bugfix and breaking change. I'll add a regular changes entry.